lib/cm_admin/model.rb

```diff
@@ -152,8 +162,19 @@ def field(field_name, options={})
       @available_fields[:show] << CmAdmin::Models::Field.new(field_name, options)
     end
 
-    def form_field(field_name, options={})
-      @available_fields[@current_action.name.to_sym] << CmAdmin::Models::FormField.new(field_name, options[:input_type], options)
+    def form_field(field_name, options={}, arg=nil)
+      if @current_action.is_nested_field == false
```

Comment:
@anbublacky - Can we change this to `unless`?
```suggestion
      unless @current_action.is_nested_field
```